pkg/volume/azure_file/azure_file.go

```diff
@@ -48,6 +48,10 @@ var _ volume.PersistentVolumePlugin = &azureFilePlugin{}
 
 const (
 	azureFilePluginName = "kubernetes.io/azure-file"
+	fileModeName        = "file_mode"
+	dirModeName         = "dir_mode"
+	defaultFileMode     = "700"
```

Comment:
`defaultFileMode` is used only in tests. Why declaring here?

Reply:
Also I suggest moving these const vars to `azure_util.go`.

Reply:
fixed